### DISCUSSION TITLE
1v1 Mode with Skill Based Match Making (SBMM/ELO)

### DISCUSSION THREAD

**Discussion by Leclowndu93150** [ppy/osu#29963] [Ideas] [answered]
Having a map pool where people can chose maps like a tournament.
And the winner gets ELO while the other  one loses elo.

**Comment by Tenexxt** [comment]
So basically playlists with a bonus scoring system.
I made a proposal in making a more of a ranked system that uses elo, tho it doesn't work on a basis of loosing elo, you only gain it like pp and gain/loose rank depending on other people. you can read my whole suggestion here: #29455

No matter what would even be considered an better option, i think the devs are sick of ranked and don't realy like the idea of expanding it for now. People constantly want pp changes (which are comming this/next month btw), so i think after they push those updates they will look at more existing things that need adjusting (like the editor, skinning, and making more mods ranked).

**Reply by Leclowndu93150** [reply]
"i think the devs are sick of ranked"

i couldn't agree more.

but what i'm talking about is playing against players that are actually the same level as you.
not playing against myself and imaginary ranks

**Comment by vatei** [comment]
I believe smoogi was experimenting with something like that, but I also remember hearing it required some significant multiplayer changes, so I don't know how far that went.

**Comment by Givikap120** [comment, marked as answer]
This is planned, but very far away
